cmd/crates/soroban-test/tests/fixtures/test-wasms/custom_account/Cargo.toml

```
@@ -0,0 +1,18 @@
+[package]
+name = "test_custom_account"
+version = "21.0.0-rc.1"
+authors = ["Stellar Development Foundation <info@stellar.org>"]
+license = "Apache-2.0"
+edition = "2021"
+publish = false
+rust-version.workspace = true
+
+[lib]
+crate-type = ["cdylib", "rlib"]
```

```suggestion
crate-type = ["cdylib"]
```
Looks like the hello_world contract had this mistake too.